src/net/dash/index.ts

```diff
@@ -115,13 +109,20 @@ export default function(
       const data = typeof response.responseData === "string" ?
         new DOMParser().parseFromString(response.responseData, "text/xml") :
         response.responseData;
+
       const parsedManifest = dashManifestParser(data, url);
+      return loadExternalRessources(parsedManifest);
 
       function loadExternalRessources(
         parserResponse : IMPDParserResponse
       ) : IManifestParserObservable {
         if (parserResponse.type === "done") {
-          return observableOf({ manifest: parserResponse.value, url });
+          const manifest = new Manifest(parserResponse.value, {
+            representationFilter: options.representationFilter,
```

Comment:
Maybe you should directly give options as an argument.

Reply:
Yep I hesitated, I have nothing against that. Will do